Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -204,6 +204,26 @@ def ap_find_tests(bld, use=[]):
             cxxflags=['-Wno-undef'],
         )
 
+_versions = []
+
+@conf
+def ap_version_append(ctx, k, v):
+    ctx.env['AP_VERSION_ITEMS'] += [(k, os.environ.get(k, v))]
+
+@conf
+def ap_version_append_str(ctx, k, v):
+    ctx.env['AP_VERSION_ITEMS'] += [(k, '"{}"'.format(os.environ.get(k, v)))]
```

Comment:
It wasn't working when the var was coming from environment. Then I figured we were gaining almost nothing for sharing the functions (and calling environ.get() twice). See https://semaphoreci.com/lucasdemarchi/ardupilot/branches/pr-waf-git/builds/1